src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesSearchEngine.cs

```diff
@@ -284,8 +280,6 @@ private async Task ProcessProjectAsync(Project project, ImmutableArray<ISymbol>
             }
             finally
             {
-                FindReferenceCache.Stop(model);
```

Comment:
no need to start/stop the cache.  the cache is created on demand and associated with the semantic model.  it is kept alive as long as the semantic model is (the lifetime of this method).